luminance/examples/12-displacement-map.rs

```diff
@@ -0,0 +1,171 @@
+//! This program is a showcase to demonstrate how you can use grayscale textures to displace the
+//! lookup location of a texture. This is commonly referred to as "displacement mapping." Here we
+//! demonstrate using multiple displacement maps to offset the lookup in different directions. The
+//! displacement also uses time as an input, so the displacement changes according to a sine wave.
+//!
+//! The texture path is read from the command line interface and is the sole argument.
+//!
+//! The image is stretched to match the window size, but the displacement maps are tiled and true to
+//! pixel size regardless of the window size.
+//!
+//! Press <w> or <s> to increase or decrease the scale factor of the displacement, respectively.
+//! Press <escape> to quit or close the window.
+//!
+//! > Note: for this example, it is recommended to compile with --release to speed up image loading.
```

Comment:
That’s a good point I should put in the texture example too. Also, now that the wiki exists and that there are a good set of examples, maybe putting numbers in front of examples file names is not really required anymore. I’m going off topic.

Thanks for this line. :)